stl/geometry/text.go

```diff
@@ -79,11 +79,12 @@ func Create3DText(username string, year string, innerWidth, baseHeight float64)
 		contextWidth:  usernameContextWidth,
 		contextHeight: usernameContextHeight,
 		fontSize:      usernameFontSize,
+		alignment:     gg.AlignLeft,
 	}
 
 	yearConfig := textRenderConfig{
 		renderConfig: renderConfig{
-			startX:     innerWidth * yearPosition,
+			startX:     innerWidth*(1-textPadding) - 75,
```

Comment:
By chosing `75`, It exaclty aligns with right side of the cuboid. 

The number was chosen by hit-and-trial method.